src/Store/CommandStoreConnector.re

```diff
@@ -106,6 +106,12 @@ let createDefaultCommands = getState => {
         ~action=CopyActiveFilepathToClipboard,
         (),
       ),
+      Command.create(
+        ~category=Some("System"),
+        ~name="Add Oni2 to System PATH", /* TODO: This should be dynamic (Add/Remove) */
```

Comment:
One thing that is a bit confusing is the script actually toggles the link, but the command is called 'Add'.

Perhaps we could just make the script always-add, and handle toggling here later? (We could add a 'Remove Oni2 from System Path' with an enabled check of seeing if the symlink exists, and augment our enabled check for 'Add' to check that the symlink _doesn't_ exist - but I'm OK deferring that).